xarray/tests/test_accessor_dt.py

```diff
@@ -248,7 +248,7 @@ def test_dask_accessor_method(self, method, parameters) -> None:
         assert_equal(actual.compute(), expected.compute())
 
     def test_seasons(self) -> None:
-        dates = pd.date_range(start="2000/01/01", freq="M", periods=12)
+        dates = pd.date_range(start="2000/01/01", freq="ME", periods=12)
```

Comment:
I know this emits a deprecation warning, but it causes a failure with our minimum pandas version.  There are several other places where similar deprecation warnings are emitted in the tests.  Let's maybe punt that until we address those all at once (I see @max-sixty is already ahead of me on this!).
```suggestion
        dates = pd.date_range(start="2000/01/01", freq="M", periods=12)
```